nekoyume/Assets/_Scripts/UI/Widget/Popup/StakingPopup.cs

```diff
@@ -0,0 +1,107 @@
+using System.Linq;
+using Nekoyume.Game.Controller;
+using Nekoyume.Helper;
+using Nekoyume.Model.Item;
+using Nekoyume.State;
+using Nekoyume.UI.Module;
+using TMPro;
+using UniRx;
+using UniRx.Triggers;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Nekoyume.UI
+{
+    public class StakingPopup : PopupWidget
+    {
+        [SerializeField] private Button closeButton;
+
+        [SerializeField] private ConditionalButton stakingButton;
+
+        [SerializeField] private Image stepImage;
+
+        [SerializeField] private TextMeshProUGUI stepText;
+
+        [SerializeField] private TextMeshProUGUI stepScoreText;
+
+        [SerializeField] private StakingItemView[] regularItemViews;
+
+        [SerializeField] private GameObject tooltip;
+
+        [SerializeField] private RectTransform tooltipRectTransform;
+
+        [SerializeField] private TextMeshProUGUI tooltipText;
+
+        private const string StepScoreFormat = "{0}<size=18><color=#A36F56>/{1}</color></size>";
+
+        protected override void Awake()
+        {
+            base.Awake();
+
+            stakingButton.OnSubmitSubject.Subscribe(_ =>
+            {
+                // Do Something
+                AudioController.PlayClick();
+            }).AddTo(gameObject);
+
+            closeButton.onClick.AddListener(() =>
+            {
+                Close();
+                AudioController.PlayClick();
+            });
+
+            tooltip.SetActive(false);
+
+            this.UpdateAsObservable()
+                .Where(_ => tooltip.activeSelf && Input.GetMouseButtonDown(0))
+                .Subscribe(_ => tooltip.SetActive(false));
+        }
+
+        public override void Show(bool ignoreStartAnimation = false)
+        {
+            var level = States.Instance.StakingLevel;
+            var deposit = States.Instance.StakedBalanceState?.Gold.MajorUnit ?? 0;
+            var regularSheet = Game.Game.instance.TableSheets.StakeRegularRewardSheet;
```

Comment:
`Game.Game.instance.TableSheets` is same with `TableSheets.Instance`.

Reply:
I applied this, Thank you!